app/src/main/cpp/skyline/kernel/svc.cpp

```diff
@@ -11,209 +11,250 @@
 
 namespace skyline::kernel::svc {
     void SetHeapSize(const DeviceState &state) {
-        u32 size{state.ctx->gpr.w1};
+        u64 size{state.ctx->gpr.w1};
```

Comment:
why u64? considering it's a w reg you should use 32

Reply:
Whoops, thanks for catching that